packages/sdks/src/functions/get-content/generate-content-url.ts

```diff
@@ -15,14 +15,19 @@ export const generateContentUrl = (options: GetContentOptions): URL => {
     apiKey,
     includeRefs = true,
     locale,
+    apiVersion = 'v2',
   } = options;
 
   if (!apiKey) {
     throw new Error('Missing API key');
   }
 
+  if (!['v1', 'v2', 'v3'].includes(apiVersion)) {
+    throw new Error('Invalid apiVersion value');
+  }
```

Comment:
Awesome that you did this! Good call, useful for users that don't rely on TypeScript.

However, since the `packages/sdks` does not support v1, I think we can remove `v1` from this list:

```suggestion
  if (!['v2', 'v3'].includes(apiVersion)) {
    throw new Error('Invalid apiVersion value');
  }
```

and add a test that throws for `v1`